community/cypher/cypher/src/test/scala/org/neo4j/cypher/ExecutionEngineTestSupport.scala

```diff
@@ -76,25 +65,32 @@ trait ExecutionEngineTestSupport extends CypherTestSupport with TestFriendlyExec
   }
 }
 
-trait TestFriendlyExecutionEngine {
+trait ExecutionEngineHelper {
+  self: GraphIcing =>
 
-  implicit class RighExecutionEngine(engine: ExecutionEngine) {
+  def graph: GraphDatabaseCypherService
 
-    def execute(query: String): ExtendedExecutionResult = {
-      engine.execute(query, Map.empty[String, Any], QueryEngineProvider.embeddedSession)
-    }
```

Comment:
Just curious; was there any particular reason for removing this function and changing all callers to send in an empty map?